.github/workflows/security.yml

```
@@ -0,0 +1,21 @@
+name: Security Checks
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)